pkg/apis/v1alpha1/provider.go

```diff
@@ -53,7 +53,7 @@ type LaunchTemplate struct {
 	// NOTE: This field is for specifying a custom launch template and is exposed in the Spec
 	// as `launchTemplate` for backwards compatibility.
 	// +optional
-	LaunchTemplateName *string `json:"launchTemplate,omitempty"`
+	LaunchTemplateName *string `json:"launchTemplate,omitempty" hash:"ignore"`
```

Comment:
Any reason not to drift on this if someone changes the name here? If someone changes the name, then we know that they have definitely drifted.

Reply:
Since it is a deprecated field, I was thinking we should not build any new capability to it

Reply:
Yeah, that's probably the right call. SGTM 👍